.bazelrc

```
@@ -4,8 +4,11 @@ build:platform_build --incompatible_enable_cc_toolchain_resolution --crosstool_t
 build:bbb-platform --config=platform_build --config=adaptive --platforms=//bazel/platforms:bbb
 build:adaptive -c opt --copt=-O3
 
+build:ut --define build_mode=ut
+test:ut --define build_mode=ut
+
 build --action_env=BAZEL_CXXOPTS="-std=c++20" --config=adaptive
-test --test_output=all
+test --test_output=all --config=ut
```

```suggestion

```
in integration test we need normal version 
```suggestion
test --test_output=all
```